proprietary/design-tokens/build/build-tokens.mjs

```diff
@@ -0,0 +1,242 @@
+import { permutateThemes as permutateTsThemes, registerTransforms } from '@tokens-studio/sd-transforms';
+import { transform } from '@divriots/style-dictionary-to-figma';
+import { readFile } from 'node:fs/promises';
+import StyleDictionary from 'style-dictionary';
+import { IMPORTED_SRC_FOLDER, DIST_FOLDER } from './constants.mjs';
+import jsonListFormat from './json-list-formatter.mjs';
+
+/** 
+ * Transforms spacing tokens with math symbols to css calc()-values
+ * NOTE: because the way references work in StyleDictionary/TokenStudio it will not transform e.g. `5 * {space.cowboy}`
+ */
+StyleDictionary.registerTransform({
+  name: 'spacing/calc',
+  type: 'value',
+  transitive: true,
+  matcher: function(token) {
+    const {type, value, isSource} = token
+    return type === 'spacing' && isSource && value.match(/[\+\-\*\/]/g);
```

Comment:
```suggestion
    const REGEX_ARTIHMETIC_OPERATORS = /[\+\-\*\/]/g; // Ergens in ./contastants.mjs , natuurlijk
    return type === 'spacing' && isSource && value.match(REGEX_ARITHMETIC_OPERATORS);
```

Triviaal. Niet per se nodig

Reply:
Als we 'm nog ergens anders (gaan) gebruiken, dan kunnen we 't lostrekken.